avm/utilities/pipelines/platform/Set-AvmGitHubIssueOwnerConfig.ps1

```diff
@@ -1,67 +1,3 @@
-<#
-.SYNOPSIS
-Parses AVM module CSV file
-
-.DESCRIPTION
-Depending on the parameter, the correct CSV file will be parsed and returned a an object
-
-.PARAMETER ModuleIndex
-Type of CSV file, that should be parsed ('Bicep-Resource', 'Bicep-Pattern')
-
-.EXAMPLE
-Next line will parse the AVM Bicep modules
-Get-AvmCsvData -ModuleIndex 'Bicep-Resource'
-
-#>
-Function Get-AvmCsvData {
```

Comment:
If removed here, please ensure it's loaded in the beginning of the script (and not in the pipeline where no reference can be found ;) )

Reply:
which reference can't be found?

Reply:
A reference for the invocation of `Get-AvmCsvData`. In the pipeline only one script is invoked and only one script should be loaded. As per the above replies - make sure set the `Set-AvmGitHubIssueOwnerConfig` function loads the `Get-AvmCsvData` function somewhere in the beginning - as opposed to it's calling pipeline.

Reply:
# Suggestion

Current:
```PowerShell
function Set-AvmGitHubIssueOwnerConfig {
  [CmdletBinding(SupportsShouldProcess)]
  param (
    [Parameter(Mandatory = $true)]
    [string] $Repo,
    [Parameter(Mandatory = $true)]
    [string] $IssueUrl
  )
  $issue = gh issue view $IssueUrl.Replace('api.', '').Replace('repos/', '') --json 'author,title,url,body,comments' --repo $Repo | ConvertFrom-Json -Depth 100

  # (...)
}
```

Alternative:
```PowerShell
function Set-AvmGitHubIssueOwnerConfig {

  [CmdletBinding(SupportsShouldProcess)]
  param (
    [Parameter(Mandatory = $true)]
    [string] $Repo,

    [Parameter(Mandatory = $true)]
    [string] $IssueUrl,

    [Parameter(Mandatory = $false)]
    [string] $RepoRoot = (Get-Item -Path $PSScriptRoot).parent.parent.parent.parent.FullName
  )

  # Loading helper functions
  . (Join-Path $RepoRoot 'avm' 'utilities' 'pipelines' 'platform' 'Get-AvmCsvData.ps1')

  $issue = gh issue view $IssueUrl.Replace('api.', '').Replace('repos/', '') --json 'author,title,url,body,comments' --repo $Repo | ConvertFrom-Json -Depth 100

  # (...)
}
```

Reply:
done